src/metrics.ts

```diff
@@ -182,33 +182,53 @@ export const MetricsProcessor = (
     };
   };
 
-  const _send = (): void => {
+  const _send = async () => {
```

Comment:
Might as well keep a return type
```suggestion
  const _send = async (): Promise<void> => {
```